tests/test_objective_funs.py

```diff
@@ -1122,6 +1124,70 @@ def test_target_profiles():
     )
 
 
+@pytest.mark.unit
+def test_signed_plasma_vessel_distance():
```

Comment:
can this be moved to the `TestObjectiveFunction` class to keep things organized?